### PR TITLE
Issue fixed: Now we get topic list for the input stack

### PR DESCRIPTION
**Pull request by jyasveer**

### CODE REVIEW

**Comment by jyasveer** [comment]
Issue https://github.com/openshiftio/openshift.io/issues/735 is fixed now.

File: bayesian/api_v1.py [hunk 1]
```diff
@@ -557,7 +554,7 @@ class StackAnalyses(ResourceWithSchema):
 
     @staticmethod
     def post():
-        decoded = decode_token()
+        decoded = {}
```
Comment (miteshvp) [inline bayesian/api_v1.py]: what will this change do?

File: bayesian/api_v1.py [hunk 2]
```diff
@@ -391,6 +384,10 @@ def get(external_request_id):
                         "overall_score": 0,
                         "magnitude": 0
                     }
+                
+                # Adding topics from the recommendations
+                stack_recommendation = get_item_from_list_by_key_value(recommendations, "manifest_file_path", stack.get("manifest_file_path"))
```
Comment (miteshvp) [inline bayesian/api_v1.py]: where is `get_item_from_list_by_key_value` implemented? Have you missed adding any files?

File: bayesian/api_v1.py (hunk 1 above)
Comment (jyasveer) [reply bayesian/api_v1.py]: sorry. this is a mistake. i had to change this to run in my local setup

File: bayesian/api_v1.py (hunk 2 above)
Comment (jyasveer) [reply bayesian/api_v1.py]: this method is already present in utils.py

**Comment by miteshvp** [comment]
[test this]

**Comment by miteshvp** [comment]
[test]

**Pull request merged**